Package.swift

```diff
@@ -3,6 +3,16 @@
 
 import PackageDescription
 
+let platformsWithThreads: [Platform] = [
+    .iOS,
+    .macOS,
+    .tvOS,
+    .watchOS,
+    .macCatalyst,
+    .driverKit,
+    .android,
+    .linux,
```

Comment:
Does this intentionally exclude Windows? This would be a change in behavior for that platform (unless this is already an issue today for that platform)

Reply:
That was accidental, fixed now.